### PR TITLE
debian/control.docs.in - invitations to Weblate

### PR DESCRIPTION
**Pull request by smoe**
More verbose description of why the documentation of LinuxCNC is important.
Added invitations to contribute to Weblate.
Added translation for French documentation (@silopolis, please check, also have a look at the extension of the Spanish, please).

I have intentionally not translate the part mentioning Weblate. Maybe I should have done.

### CODE REVIEW

File: debian/control.docs.in [hunk 1]
```diff
@@ -37,11 +58,22 @@ Replaces: linuxcnc-uspace (<= 2.9.0~pre0+git20220402.2500863908-4)
...
+ dispositivo de este tipo, ya que la tecnología descrita es transferible a
+ configuraciones automatizadas arbitrarias, desde el control de la temperatura
+ a través de puertas de garaje hasta cualquier nivel de procesamiento computacional
+ de la entrada sensorial y el control de los actuadores que sea necesario.
+ .
+ The documentation was recently prepared for computer-aided translations to all other
+ languages on Weblate (https://hosted.weblate.org/projects/linuxcnc/linuxcnc-docs/).
+ We need help with the manual confirmation o the automated transition of the earlier
+ manual Spanish translation, for which a passive understanding of the language
+ is sufficient. Please help if you can.
```
Comment (silopolis) [inline debian/control.docs.in:76]:
```suggestion
La documentación se preparó recientemente para su traducción asistida por computadora a todos
los demás idiomas en Weblate (https://hosted.weblate.org/projects/linuxcnc/linuxcnc-docs/).
Necesitamos ayuda con la confirmación manual de la transición automatizada de la traducción
manual anterior al español, para lo cual una comprensión pasiva del idioma es suficiente.
Por favor ayuda si puedes.
```

File: debian/control.docs.in [hunk 2]
```diff
@@ -51,11 +83,17 @@ Recommends: xdg-utils
 Suggests: pdf-viewer
 Conflicts: linuxcnc-doc-cn
 Replaces: linuxcnc-doc-cn
-Description: motion controller for CNC machines and robots (Chinese
- documentation)
- LinuxCNC is the next-generation Enhanced Machine Controller which
- provides motion control for CNC machine tools and robotic
+Description: motion controller for CNC machines and robots (Chinese documentation)
```
Comment (silopolis) [inline debian/control.docs.in:86]:
```suggestion
Description: Motion controller for CNC machines and robots (Chinese documentation)
```

**Review by silopolis — CHANGES_REQUESTED** [review]
Changed first character of descriptions to capital as it seems appropriate for a "Property: Value" pair.
Translated the remaining paragraphs for french and spanish

**Comment by smoe** [comment]
Squashed everything, line-wrapped, and rebased to master. Fingers crossed, this builds now.

**Comment by smoe** [comment]
It builds. @SebKuzminsky or @jepler, may I ask either of you to have an extra eye on this one?

**Comment by smoe** [comment]
Rebased to master (no conflicts), added missing period in top.in, bumped standard to 4.6.1.

**Comment by smoe** [comment]
Rebased to master because of a tiny conflict.

I just saw that I commited 
[docs: More translation-motivated changes](https://github.com/LinuxCNC/linuxcnc/pull/1961/commits/ffd6a1b89aeb336fa141b5d51e828ccee9c6ae9e)
[ffd6a1b](https://github.com/LinuxCNC/linuxcnc/pull/1961/commits/ffd6a1b89aeb336fa141b5d51e828ccee9c6ae9e) to the wrong branch. Those extra changes look sufficiently harmless and they are in a separate commit such that I hope that you allow these to remain in this PR.

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> Those extra changes look sufficiently harmless and they are in a
> separate commit such that I hope that you allow these to remain in
> this PR.

I have no opinion there, but ping me on IRC if you want help moving
those to a separate branch instead.  I can help you with the 'git
checkout -b', 'git reset' and 'git push -f' recipe if you need it.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by smoe** [comment]
If agreeing with the individual commits, please just accept it.

**Comment by smoe** [comment]
Translations should likely go elsewhere, no pointer to weblate.

**Pull request closed without merging**